.changeset/great-geese-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"@preact/signals-react": patch
+---
+
+Use isomorphic layout effect hook
```

Comment:
Thanks!

We should add better docs here (I now realize this is completely missing from our [contributing doc](https://github.com/preactjs/signals/blob/main/CONTRIBUTING.md)), but the general idea is that this makes up our change log when we publish a new version.

As such, I'd suggest this as it makes it a bit clearer to users how it affects them:

```suggestion
Silences noisy warnings about `useLayoutEffect` whilst using SSR by switching to an isomorphic layout effect hook
```